src/azul/chalice.py

```diff
@@ -123,6 +124,23 @@ def aws_name(self) -> str:
 C = TypeVar('C', bound='AppController')
 
 
+def handle_uncaught_exception(self: RestAPIEventHandler) -> Response:
+    request = self.current_request
+    path = getattr(request, 'path', 'unknown')
+    log.error('Caught exception for path %r', path, exc_info=True)
+    body = {
+        'Code': 'InternalServerError',
+        'Message': 'An internal server error occurred.',
+        'RequestId': request.lambda_context.aws_request_id
+    }
+    return Response(body=body, status_code=500)
+
+
+# Ensures that stacktraces are never prevent in the response, even when
+# `app.debug` is set to true
```

Comment:
```suggestion
# Ensures that stacktraces are never printed in the response, even when
# `app.debug` is set to true
```

… maybe?

Reply:
I had meant to type "present" but I decided to use more active language instead.